src/Microsoft.Framework.Runtime.Interfaces/IRuntimeEnvironment.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Framework.Runtime
+{
+    public interface IRuntimeEnvironment
+    {
+        OperatingSystem OperatingSystem { get; }
+        
+        Version OperatingSystemVersion { get; }
+
+        FrameworkName Framework { get; }
```

Comment:
This isnt right. It should be flavor and it should say what CLR you're running on not what framework.